src/br/features/evolve.py

```diff
@@ -474,10 +476,10 @@ def get_evolution_dict(
                         if len(intermediate_embed.shape) > 2:
                             if fit_pca:
                                 intermediate_embed = pca.transform(
-                                    intermediate_embed[:, 1:, :].mean(axis=1)
+                                    np.linalg.norm(intermediate_embed, axis=-1)
                                 )
                             else:
-                                intermediate_embed = intermediate_embed[:, 1:, :].mean(axis=1)
+                                intermediate_embed = np.linalg.norm(intermediate_embed, axis=-1)
```

Comment:
This seems like a substantive change? Can you give more context?

Reply:
this -> `intermediate_embed[:, 1:, :].mean(axis=1)` was specifically for MAE features (batch X tokens X features). The output of that model has the class token as the first dimension, and so we take the mean across every dimension except the first. 

this -> `np.linalg.norm(intermediate_embed, axis=-1)` is for vector neuron features (rotation equivariant point cloud encoder outputs -  batch x feature x 3). Its a vector output so we take the vector norms to get the features. 

We dont really use MAE in most of the paper analysis, so this is removed currently